test/metrics/maxFileDescriptorsTest.js

```diff
@@ -1,29 +1,37 @@
 'use strict';
 
 const exec = require('child_process').execSync;
+const Registry = require('../../index').Registry;
 
-describe('processMaxFileDescriptors', () => {
+describe.each([
+	{ tag: 'Prometheus', regType: Registry.PROMETHEUS_CONTENT_TYPE },
+	{ tag: 'OpenMetrics', regType: Registry.OPENMETRICS_CONTENT_TYPE },
+])('processMaxFileDescriptors with $tag registry', ({ tag, regType }) => {
 	const register = require('../../index').register;
 	const processMaxFileDescriptors = require('../../lib/metrics/processMaxFileDescriptors');
 
 	beforeAll(() => {
 		register.clear();
 	});
 
+	beforeEach(() => {
+		register.setContentType(regType);
+	});
+
 	afterEach(() => {
 		register.clear();
 	});
 
 	if (process.platform !== 'linux') {
-		it('should not add metric to the registry', async () => {
+		it('should not add metric to the $tag registry', async () => {
```

Comment:
this is not interpolated, you need to do something like

```suggestion
		it(`should not add metric to the ${tag} registry`, async () => {
```

manually 🙂